.github/workflows/actions.yml

```diff
@@ -60,15 +60,15 @@ jobs:
           - java-version: 17
             buildcmd: ci/test-mill-dev.sh
           - java-version: 17
-            buildcmd: ./mill -i -k "example.__.test"
+            buildcmd: ./mill -i -k "example.__.local.test"
```

Comment:
The examples folder has grown pretty big, and they're usually pretty straightforward. Doesn't make sense to run each one three times in local/fork/server configs